docs/modules/ROOT/pages/howtos/quickstart_aks.adoc

```diff
@@ -10,13 +10,16 @@
 - Access to GitLab or GitHub (only supported CI/CD for now),
 - Knowledge of https://terraform.io[Terraform] basics
 
-== Create your Terraform root module
+== Create your Terraform files
 
 Camptocamp's DevOps stack is a Terraform composition module
-that has to be instantiated from the Terraform root module.
+has to be instantiated from the Terraform root module.
```

Comment:
Seems to me that this was actually the correct version, and all others were wrong.

Reply:
updated in latest commit